clients/src/main/resources/common/message/LeaderChangeMessage.json

```diff
@@ -22,7 +22,9 @@
     {"name": "LeaderId", "type": "int32", "versions": "0+",
       "about": "The ID of the newly elected leader"},
     {"name": "Voters", "type": "[]Voter", "versions": "0+",
-      "about": "The voters who voted for the current leader"}
+      "about": "The voters who voted at the time of election"},
+    {"name": "GrantingVoters", "type": "[]Voter", "versions": "0+",
+      "about": "The voters who voted for the leader at the time of election"}
```

Comment:
I think we should add an `LeaderEpoch` field that documents the epoch associated with this `LeaderChangeMessage`. What do you think @vamossagar12 and @hachikuji ?

Reply:
Yeah I think it makes sense to add the LeaderEpoch.

Reply:
Plz let me know if this needs to be added.